ambassador/tests/009-rate-limit/config/mapping-qotm.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: ambassador/v0
+kind: Mapping
+name: qotm_mapping
+prefix: /qotm/
+service: qotm
+---
+apiVersion: ambassador/v0
+kind: Mapping
+name: qotm_rate_limit_mapping
+prefix: /qotm-rate/
+service: qotm
+rate_limits: [{}]
+---
+apiVersion: ambassador/v0
+kind: Mapping
+name: qotm_rate_limit_config_mapping
+prefix: /qotm-rate-custom/
+service: qotm
+rate_limits:
+  - {}
+  - descriptor: custom-label
```

Comment:
I'm seeing this empty dict a lot; what's the purpose?

If the idea is to test that the code is rejecting a ratelimit object that doesn't have all the necessary data, let's just make the object fail validation if it doesn't have everything, and only have one bad dict that's clearly commented as "this is testing a failure case" so people using the tests as tutorials don't get confused.

If the purpose is something else, clearly I'm confused and need to be enlightened. :)

Reply:
OK, maybe have the empty dict in just one test, with a comment explaining what's going on? We've been finding that people do indeed read the tests as examples. 😄

Reply:
Sure, adding descriptive comments to the test case is a good idea :)